# Clearing human bone marrow reveals spatial heterogeneity of the stromal niche

This repository contains the information on the code related to the studies conducted for this manuscript.

The bone marrow stroma (BMS) is among the most enigmatic parts of the bone. It contains skeletal stem cells and committed progenitors for the osteo- and adipo-lineages, supports hematopoiesis, and regulates the activity of leukemia cells1,2. Those stromal cells expressing C-X-C-motif-12 chemokine (CXCL12) are unequivocal for hematopoiesis and are often called CXCL12-abundant reticular (CAR) cells3. The microenvironments, or niches where CAR cells are located, determine their functionality4. However, these niches are largely unknown in humans. To fill this gap of knowledge, we developed a tissue-clearing protocol allowing detection and spatial quantification of mRNAs and proteins in sizeable human bone biopsies, named DeepBone and combined it with an atlas of bone marrow single-cell RNA sequencing (scRNAseq) and neuronal network (NN) analysis. CAR cells were found transcriptionally identical to bone marrow mesenchymal stromal cells (BM-MSCs), and all abundantly express CXCL12 chemokine. Spatial analysis revealed that the niches where CXCL12-expressing cells reside were formed by a complex composition of capillaries, sinusoids, adipocytes, and bony trabeculae, and this landscape drastically changed with age. The niche composition was heterogeneous but formed a continuum revealed by unsupervised NN. These insights into spatial organization of the human bone microenvironment will improve our understanding of the mechanisms underlying numerous hematological and bone diseases.

## Single-cell data availability
[CellxGene bone marrow atlas](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcellxgene.cziscience.com%2Fcollections%2F0391c84c-d57d-4741-9277-e4d58f9a3d0c&data=05%7C02%7Ckriukov.e%40northeastern.edu%7C17cfa576b574406326dc08dc421aba17%7Ca8eec281aaa34daeac9b9a398b9215e7%7C0%7C0%7C638457929777265718%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=k2SELErdM4pfNXlVEfra1Hht9oqJINEFSTiu3QVy3L8%3D&reserved=0) <br />
For the raw data and/or .h5ad/.h5Seurat object, please contact Ostap Dregval or Emil Kriukov.

## Main code developers
1. Ostap Dregval - Bioinformatician, Andrei Chagin Lab, Gothenburg University | ostap.dregval@gu.se |  Python Analysis of Single-cell Data
2. Emil Kriukov - Senior Bioinformatician, Petr Baranov Lab, Schepens Eye Research Institute of Mass Eye and Ear, Harvard Medical School | emilprmu@gmail.com ekriukov@meei.harvard.edu kriukov.e@northeastern.edu | Light-sheet Images Python Analysis, Azimuth
3. Tsz Long Chu - Researcher, Andrei Chagin Lab, Karolinska Institutet | tsz.long.chu@ki.se | 3D Contour
4. Yu-wei Chang - PhD Candidate, Giovanni Volpe Lab, Gothenburg University | yu-wei.chang@physics.gu.se
5. Giovanni Volpe - Professor, Giovanni Volpe Lab, Gothenburg University | giovanni.volpe@physics.gu.se

## Corresponding authors
1. Tsz Long Chu - Researcher, Andrei Chagin Lab, Karolinska Institutet | tsz.long.chu@ki.se
2. Andrei Chagin - Professor, Andrei Chagin Lab, Gothenburg University | andrei.chagin@gu.se
3. Yu-wei Chang - PhD Candidate, Giovanni Volpe Lab, Gothenburg University | yu-wei.chang@physics.gu.se
4. Giovanni Volpe - Professor, Giovanni Volpe Lab, Gothenburg University | giovanni.volpe@physics.gu.se
5. Emiliano Gomez Ruiz - PhD Candidate, Giovanni Volpe Lab, Gothenburg University | emiliano.gomez@physics.gu.se
